packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -121,6 +153,25 @@ void main() {
     expect(box.size.height - oldHeight, greaterThanOrEqualTo(100.0)); // 100 + some margin
   });
 
+  testWidgets('ExpansionPanelList does not merge header when canTapOnHeader is false', (WidgetTester tester) async {
```

Comment:
This test doesn't really test that semantics nodes do not get merged. You probably want a test that actually looks at the semantics tree. 

Example Tests that do that:
* https://github.com/flutter/flutter/blob/master/packages/flutter/test/cupertino/action_sheet_test.dart#L825
* https://github.com/flutter/flutter/blob/master/packages/flutter/test/material/checkbox_test.dart#L60